### PR TITLE
Spacing amends

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

**Comment by discodavey** [comment]
Last changes from @chugginselifesciences. @de-code Can you check code and let me know if ok. Then merge and push live please.

**Comment by de-code** [comment]
Just linking the corresponding issue (and comment): https://github.com/elifesciences/issues/issues/5699#issuecomment-652328171

**Comment by de-code** [comment]
Screenshot before:

image


Screenshot of what it looks like with the change:

image

> Add a left margin of 48px to the logo below 1200px screen width.

I can't judge whether the margin change is as intended. Would we not want the logo to align with the main text?

> Can we also remove the -1.6% left and right margin on the .grid?

I am not sure whether that has changed? It looks as wide as before.

> Can you remove the s from examples in the first `<p>` in the examples section please?
> I think it should be “Convert example articles”, not “Convert examples articles”, so looks like a typo

The typo doesn't seem to have been addressed yet.

> Then I think we should remove the `<br>` from the second `<p>` after all and change the last sentence to

To me breaking after `The other` seems less readable but of course that changes with a smaller window size. The screenshot in the issue comment is narrower. So maybe that is because of the point above, should it be less wide?

**Comment by discodavey** [comment]
@de-code Can you have one final look. I paired with @chugginselifesciences to get the design and alignment to how he wants it. If all ok can you merge and push please?

**Comment by de-code** [comment]
Okay, just including screenshots for the record..

On a wide browser:

localhost_8000_

Somewhat reduces width:

localhost_8000_ (1)

Narrow width:

localhost_8000_ (2)

**Review by de-code — APPROVED** [review]

File: demo/src/styles/patterns/organisms/site-header.scss [hunk 1]
```diff
@@ -114,3 +114,49 @@
...
+  }
+}
+
+@media only all and (min-width: #{get-rem-from-px($bkpt-site--x-wide)}em) {
+
+  .site-header__logo_link {
+    margin: 4px 0 0 -4px;
+
+  }
+}
```
Comment (de-code) [inline demo/src/styles/patterns/organisms/site-header.scss]:
```suggestion
}

```

**Pull request merged**